tests/mocked_plans/sample_plans/018346c5-f95c-46df-a34e-9a78bdec0018.yaml

```diff
@@ -90,6 +126,7 @@ plan: |
                     Ψ Upgrade software packages on unit 'ceph-mon/1'
                     Ψ Upgrade software packages on unit 'ceph-mon/2'
                 WARNING: Changing 'ceph-mon' channel from latest/stable to octopus/stable. This may be a charm downgrade, which is generally not supported.
+                Wait for up to 2400s for model '018346c5-f95c-46df-a34e-9a78bdec0018' to reach the idle state
```

Comment:
I think that the wait time here shouldn't be the same as waiting for the application to upgrade the payload. I think it can be less like 300 seconds

Reply:
I don't want to make it overcomplicated; and this is just the timeout value.  It should return a lot quicker than this.